### ISSUE TITLE
[QUESTION] Migrate from v1 to v2

### ISSUE THREAD

**Issue by popy2k14** [lubeda/EspHoMaTriXv2#132]
# Question

@lubeda is there an easy way to migrate over from v1 to v2?
Running two ulanzis now with v1 with no issues but as you archived v1 i am planning to migrate to v2.

## Additional information

- used Hardware:
  - ESP32

  - EspHoMaTriXv1 version: 2023.4.0

**Comment by lubeda** [comment]
Not really! Since the possibilities changed between the two versions, you have to rethink your automations. 
Using a text editor could help to find old `add_screen` commands but the new `icon_screen` has additional parameters ("r", "g", "b").  So it is a manual process. Sorry

**Comment by Schampanja** [comment]
Hi, 
I have a homemade DIY Display by using the Tutorial from the SmartHome Yourself Youtube Chanel.
I used the fullfeature.yaml from V1 as en example to cnfigure the project.
Now I realised, that there is a V2 version of EspHoMatrix available.
Can i use my old configuration by just changing the external_component?
Or do I have to create a new configuration file? And if Yes, is there a "fullfeature" example like in V1?
I just found the Ulanzi yaml.

Thx a lot

David
